sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -515,3 +515,82 @@ ALTER TABLE `api_token` ADD `token_api` varchar(40);
 #IfMissingColumn api_token patient_id
 ALTER TABLE `api_token` ADD `patient_id` bigint(20) NOT NULL;
 #EndIf
+
+#IfNotTable fhir_forced_id
+CREATE TABLE `fhir_forced_id` (
+    `pid` bigint(20) NOT NULL,
+    `forced_id` varchar(100) NOT NULL,
+    `resource_pid` bigint(20) NOT NULL,
+    `resource_type` varchar(100) DEFAULT '',
```

Comment:
resource_type column
It would be nice if MySQL 5.x supported check constraints.
I would opt to remove the default empty string `` and define the column as NOT NULL. Would there be a reason why we would want to support an empty string value?

Reply:
nope except habits.